.changeset/chilled-pants-wonder.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/backend-openapi-utils': patch
+'@backstage/plugin-cost-insights': patch
+'@backstage/create-app': patch
```

Comment:
```suggestion
'@backstage/backend-openapi-utils': patch
```

Dropping the last two since they're just in-repo fixes and don't affect the published package